packages/dream-httpaf/dream-httpaf.1.0.0~alpha4/opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "dune" {>= "2.7.0"}  # --instrument-with.
   "gluten"
   "gluten-lwt-unix"
-  "h2" {< "0.13.0"}
+  "h2" {= "0.12.0"}
```

Comment:
Looks like this was introduced in 0.9.0:

https://github.com/anmonteiro/ocaml-h2/releases/tag/0.9.0

So I think we can try to be a bit more permissive here:

```suggestion
  "h2" {>= 0.9.0 & < "0.13.0"}
```